lib/presentation/util/extensions/datetime_extension.dart

```diff
@@ -35,4 +35,9 @@ import 'package:intl/intl.dart';
 extension DateTimeExtension on DateTime {
 
   String getMMMddyyyyFormatString() => DateFormat.yMMMd().format(this);
+
+  String get_yMMMMd_formatWithJm() => DateFormat.yMMMMd().add_jm().format(this);
```

Comment:
What is `jm`? Besides that, IMO, you should not use this format for your method name?

Reply:
`add_jm()` used for time format

Reply:
It is not answered my question? What is it? Why we need it?

Reply:
It a function for time format. Example: `DateFormat.yMd().add_jm() -> 7/10/1996 5:08 PM`

Reply:
No, we should not use this schema.